roles/pulp_common/README.md

```diff
@@ -152,6 +154,13 @@ If it is set to "packages", the following variables are used, or behave *differe
   installer will fix it.)
   Defaults to `false`.
 
+Role Variables for advanced usage
+---------------------------------
+
+* `pulp_version`: Specify a specific version of pulpcore one would like to install or upgrade to.
+   By default the installer will do the right thing by using the version of pulpcore it is designed
+   for and tested with. It is strongly advised against setting unless one knows whats he is doing.
```

Comment:
```suggestion
   for and tested with. It is strongly advised against setting unless one knows what one is doing.
```
I'd also be ok with
```suggestion
   for and tested with. It is strongly advised against setting.
```
Or a more general statement/warning like "The use of these variables requires a profound knowledge of the installer."

Reply:
Done.